server/database/config/connection.js

```diff
@@ -0,0 +1,21 @@
+const { Pool } = require('pg');
+require('dotenv').config();
+
+let dbUrl;
+
+if (process.env.NODE_ENV === 'development') {
+  dbUrl = process.env.DB_URL;
+} else if (process.env.NODE_ENV === 'production') {
+  dbUrl = process.env.DATABASE_URL;
+} else if (process.env.NODE_ENV === 'test') {
+  dbUrl = process.env.TEST_DB_URL;
+}
+
```

Comment:
Here we have one condition so i suggest to use "switch" instead of "if statement"
```suggestion
 switch (process.env.NODE_ENV) {
    case 'development':
    dbUrl = process.env.DB_URL;
    break;
   
```